digital-workspace/latest/governance/folder-rules.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Folder rules
+---
+
+In the Digital Workspace you can define folder rules to manage your content automatically, in the same way email clients use rules to filter messages and incoming spam. Rules dictate how content entering, leaving, or currently residing in a folder is managed. When you define a rule, it only applies to new content added to the folder. Items that were in the folder before the rule was defined aren't affected by it.
+
+> **Note:** Even if a folder doesn't have its own rules, it could have inherited rules from a parent folder.
+
+There are three parts to a folder rule:
+
+* The event that triggers the rule.
+* The conditions the content has to meet.
+* The action performed on the content.
+
+The events that can trigger a rule are:
+
+* A content item arrives in the folder.
+* A content item leaves the folder (it's moved or deleted).
+* A content item in the folder is modified.
+
+Here are some examples of how you can use rules to automate repetitive tasks:
+
+* All records with certain criteria that enter a folder can be filed as a record.
+* All incomplete records placed in a folder are completed.
+* A record with certain criteria that enters a folder can be linked to a category.
+* Remove the aspect of certain records that enter a folder.
```

Comment:
you are giving example of records - and we are not supporting records at the moment - only content

Reply:
Added different examples.